lte/gateway/docker/python-precommit/README.md

```diff
@@ -34,6 +34,8 @@ cd $MAGMA/lte/gateway/python
 ./precommit.py --format--diff
 ```
 
+Note: Python version >= 3.7 is needed to run the above precommit.py script
```

Comment:
I think we should be add this note in the precommit script also.

Reply:
Thank you for this suggestion @themarwhal